### PR TITLE
What's New for engine 3.13

### PR DESCRIPTION
**Pull request by micheles**
See https://github.com/gem/oq-engine/blob/whatsnew/doc/whats-new.md

### CODE REVIEW

File: doc/whats-new.md [hunk 1]
```diff
@@ -1,404 +1,480 @@
...
+to read) and the usage of numba in `get_slices`. The effect is dramatic
+in the ESHM20 model, much less so in smaller models.
+
+The documentation of the point source gridding approximation has been
+improved, with a section about how to make the Canada model 26x faster:
+
+https://docs.openquake.org/oq-engine/advanced/point-source-gridding.html
+
+We improved the precision of the `pointsource_distance` approximation:
+that means the now you can use smaller values for the `pointsource_distance`
```
Comment (mmpagani) [inline doc/whats-new.md:104]: the -> that

File: doc/whats-new.md [hunk 2]
```diff
@@ -1,404 +1,480 @@
...
+
+- [Chiara Mascandala](https://github.com/mascandola)
+  contributed a new GMPE LanzanoEtAl2019_RJB_OMO
+  and fixed a bug in the SgobbaEtAl2020 GMPE. She also contributed
+  new variations of Lanzano & Luzi (2019), Skarlatoudis et al. (2013),
+  and BCHydro GMPEs.
+
+- [Giuseppina Tusa](https://github.com/gtus23) contributed the Tusa-Langer-Azzaro (2019) GMPE.
+
+- We added support for EAS, FAS and DRVT intensity measure types, used
```
Comment (mmpagani) [inline doc/whats-new.md:225]: "Thanks to EDF we added  support ..."

File: doc/whats-new.md [hunk 3]
```diff
@@ -1,404 +1,480 @@
...
-disaggregation outputs in traditional format (i.e. Bazzurro and Cornell 1999),
-where the probabilities sum up to 1.
+We removed the dependency from python-prctl which was not needed anymore.
 
-- The command `oq --version` now gives the git hash if the engine was
-installed with the universal installer using the `--version=master`
-option.
+We removed all upper bounds on the dependencies to make it possible
+for people to install the engine with newer libraries. Thanks to that
+the the engine now unofficially works on macOS with an M1 processor by
```
Comment (mmpagani) [inline doc/whats-new.md:434]: remove one 'the'

**Review by mmpagani — COMMENTED** [review]
Thanks Michele fro the comprehensive description. I left few very minor comments.

**Comment by micheles** [comment]
All fixed in https://github.com/gem/oq-engine/blob/engine-3.13/doc/whats-new.md, thanks @mmpagani

**Pull request merged**